falcon/asgi/app.py

```diff
@@ -37,7 +39,7 @@
 __all__ = ['App']
 
 
-_EVT_RESP_EOF = {'type': 'http.response.body'}
+_EVT_RESP_EOF = {'type': EventType.HTTP_RESP_BODY}
```

Comment:
Since I was sprinkling yet another event string throughout the code (``'http.disconnect'``)